src/include/units/bits/deduced_symbol_text.h

```diff
@@ -45,7 +45,7 @@ constexpr auto operator_text()
       return basic_fixed_string("/");
     }
     else {
-      return basic_fixed_string("⋅");
+      return basic_symbol_text("⋅", ".");
```

Comment:
The official ASCII replacement for `·` character should just be a space ` `.

Reply:
Not `*`?

Reply:
No, the ISO 80000 standard explicitly says:

> When symbols for quantities are combined in a product of two or more quantities, this combination is indicated in one of the following ways: `ab, a b, a · b, a × b`.

From those, I think that space is the best solution.

Reply:
Actually looking into it we probably should add spaces on both sides of the Unicode symbol to be ` · ` rather than just `·`.

Reply:
No problem! On it.